PasteRack.rkt

```diff
@@ -0,0 +1,10 @@
+#lang racket/base
+(require browser/external)
+; Launch http://pasterack.org/ in browser
+;; string? -> (or/c string? #f)
+(provide item-callback)
+(define (item-callback str)
+  (send-url "http://pasterack.org/")
+  str)
```

Comment:
Make it return #f instead of str so that the file is not modified and I'll approve the push request.
See http://docs.racket-lang.org/script-plugin/index.html#%28part._.The__rkt_file%29